### PR TITLE
UI: Use stats trackers for resource rows

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
- Update the stat polling on alloc rows to use the new stat tracker
- Add resource tracking to task rows on the alloc detail page
- Remove dead code from the old stat polling method.

### CODE REVIEW

**Comment by DingoEatingFuzz** [comment]
I suspect that the visual issues you are seeing here and in the other PR are related to fixture data. Which I should fix, but in the meantime if you proxy to the demo site, I think things will work fine for you.

**Comment by johncowen** [comment]
Quick note, I pointed it to the demo site and set `ENV['ember-cli-mirage'].enabled` to `false` and I'm seeing things now for this PR (and I would assume the other one also). I'll see if I can spend a bit of time with it a bit later. The graphs looks sweet!

**Comment by johncowen** [comment]
Hey, these are looking cool!

The 'jumping'' thing I mentioned above has gone also, so that must have been the fixtures.

Semi-related thing I noticed is that when you click away from the Allocation page (the one that has the graphs and the 'Task' table) using the second to last breadcrumb, you hit the following:

screen shot 2018-10-01 at 14 12 02

It's pretty consistent for me, I've tried in a few other places and it doesn't error (but I did find another one which seems unrelated), it seems to only be clicking the second to last breadcrumb.

Cheers,

John

**Comment by DingoEatingFuzz** [comment]
I was able to reproduce the bug you observed and tracked it down to a bad namespace lookup for that particular route.

I'd prefer to fix that bug in a different PR though since this work isn't what introduced the issue.

File: ui/app/components/task-row.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  tagName: 'tr',
+  classNames: ['task-row', 'is-interactive'],
+
+  task: null,
+
+  // Internal state
+  statsError: false,
+
+  enablePolling: computed(() => !Ember.testing),
```
Comment (johncowen) [inline ui/app/components/task-row.js]: Couple of question on this for my own benefit really:

1. Could you use `alias` for this?
2. Does `Ember.testing` ever change once things are running?

**Comment by johncowen** [comment]
> I'd prefer to fix that bug in a different PR though since this work isn't what introduced the issue

Cool yeah, sounds good, ping me when you do it and I can take a look again.

I gave it another code once over and a couple of q's arose from that just for my own benefit. 

Other than that LGTM!

File: ui/app/components/task-row.js (hunk 1 above)
Comment (DingoEatingFuzz) [reply ui/app/components/task-row.js]: Alias only works for paths on the object already (e.g., `this.Ember.testing`), so no.

I just tried it and yes, `Ember.testing` can be changed at runtime, but it never happens from framework code.

**Comment by DingoEatingFuzz** [comment]
Thanks for the review @johncowen, I added `role="tooltip"` throughout the app.

**Pull request merged**